[BUG] demo.launch will fail with Exception

**Describe the bug**
I launched demo.py but Fetch in gazebo will get stuck in the front of the table and finally fail with TypeError.

Fetch rotating in the front of the table when getting stuck
![Screenshot from 2020-12-01 22-19-56](https://user-images.githubusercontent.com/9410362/100745951-5df88580-3423-11eb-99b1-5f9f1e1abb5d.png)

When demo.py failling with TypeError in the end, Fetch raise its torso.
![Screenshot from 2020-12-01 22-26-42](https://user-images.githubusercontent.com/9410362/100746627-4f5e9e00-3424-11eb-9a83-7dd2dfd17e05.png)

Error which I got.
```bash
[ERROR] [WallTime: 1606829042.358124593, 367.401000000] [node:/move_base] [func:MoveBase::executeCycle]: Aborting because a valid control could not be found. Even after executing all recovery behaviors
[INFO] [WallTime: 1606829042.392132, 367.425000] [node:/demo] [func:<module>]: Raising torso...
[INFO] [WallTime: 1606829050.088133, 373.459000] [node:/demo] [func:<module>]: Picking object...
Traceback (most recent call last):
  File "/opt/ros/melodic/lib/fetch_gazebo_demo/demo.py", line 274, in <module>
    grasping_client.updateScene()
  File "/opt/ros/melodic/lib/fetch_gazebo_demo/demo.py", line 152, in updateScene
    wait = False)
TypeError: addSolidPrimitive() got an unexpected keyword argument 'wait'
[demo-7] process has died [pid 4420, exit code 1, cmd /opt/ros/melodic/lib/fetch_gazebo_demo/demo.py __name:=demo __log:=/home/sktometometo/.ros/log/504a1e90-33d7-11eb-ac21-04ed33eed89c/demo-7.log].
log file: /home/sktometometo/.ros/log/504a1e90-33d7-11eb-ac21-04ed33eed89c/demo-7*.log
```

**To Reproduce**
The steps to help us reproduce this error.
We recommend using Docker to ensure your setup is as clean as possible, or using a new catkin workspace.

My environment is Ubuntu 18.04 + ROS melodic on Thinkpad T490.
fetch_gazebo and fetch_gazebo_demo is installed with apt.
The versions are below
```bash
/opt/ros/melodic/share/fetch_gazebo $ cat package.xml 
<?xml version="1.0"?>
<package format="2">
  <name>fetch_gazebo</name>
  <version>0.9.2</version>
  <description>
    Gazebo package for Fetch.
  </description>
  <author>Michael Ferguson</author>
  <maintainer email="amoriarty@fetchrobotics.com">Alex Moriarty</maintainer>
  <maintainer email="narora@fetchrobotics.com">Niharika Arora</maintainer>
  <maintainer email="selliott@fetchrobotics.com">Sarah Elliott</maintainer>
  <license>BSD</license>
  <url>http://ros.org/wiki/fetch_gazebo</url>

  <buildtool_depend>catkin</buildtool_depend>

  <build_depend>angles</build_depend>
  <build_depend>gazebo_dev</build_depend>

  <depend>control_toolbox</depend>
  <depend>boost</depend>
  <depend>gazebo_plugins</depend>
  <depend>gazebo_ros</depend>
  <depend>robot_controllers</depend>
  <depend>robot_controllers_interface</depend>
  <depend>sensor_msgs</depend>

  <exec_depend>actionlib</exec_depend>
  <exec_depend>control_msgs</exec_depend>
  <exec_depend>depth_image_proc</exec_depend>
  <exec_depend>fetch_description</exec_depend>
  <exec_depend>gazebo</exec_depend>
  <exec_depend>image_proc</exec_depend>
  <exec_depend>nodelet</exec_depend>
  <exec_depend>rgbd_launch</exec_depend>
  <exec_depend>trajectory_msgs</exec_depend>
  <exec_depend>xacro</exec_depend>
</package>
```

```bash
/opt/ros/melodic/share/fetch_gazebo_demo $ cat package.xml 
<package>
  <name>fetch_gazebo_demo</name>
  <version>0.9.2</version>
  <description>
    Demos for fetch_gazebo package.
  </description>
  <author>Michael Ferguson</author>
  <maintainer email="amoriarty@fetchrobotics.com">Alex Moriarty</maintainer>
  <maintainer email="narora@fetchrobotics.com">Niharika Arora</maintainer>
  <maintainer email="selliott@fetchrobotics.com">Sarah Elliott</maintainer>
  <license>BSD</license>
  <url>http://ros.org/wiki/fetch_gazebo_demo</url>
  
  <buildtool_depend>catkin</buildtool_depend>

  <run_depend>actionlib</run_depend>
  <run_depend>fetch_gazebo</run_depend>
  <run_depend>fetch_moveit_config</run_depend>
  <run_depend>fetch_navigation</run_depend>
  <run_depend>moveit_commander</run_depend>
  <run_depend>moveit_python</run_depend>
  <run_depend>simple_grasping</run_depend>
  <run_depend>teleop_twist_keyboard</run_depend>
</package>
```

I have launched demo.launch in fetch_gazebo_demo and playground.launch in fetch_gazebo

```bash
~ $ roslaunch fetch_gazebo playground.launch 
... logging to /home/sktometometo/.ros/log/504a1e90-33d7-11eb-ac21-04ed33eed89c/roslaunch-Persing-3753.log
Checking log directory for disk usage. This may take a while.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

xacro.py is deprecated; please use xacro instead
started roslaunch server http://Persing:43709/

SUMMARY
========

PARAMETERS
 * /arm_controller/follow_joint_trajectory/joints: ['shoulder_pan_jo...
 * /arm_controller/follow_joint_trajectory/type: robot_controllers...
 * /arm_controller/gravity_compensation/autostart: True
 * /arm_controller/gravity_compensation/root: torso_lift_link
 * /arm_controller/gravity_compensation/tip: gripper_link
 * /arm_controller/gravity_compensation/type: robot_controllers...
 * /arm_with_torso_controller/follow_joint_trajectory/joints: ['torso_lift_join...
 * /arm_with_torso_controller/follow_joint_trajectory/type: robot_controllers...
 * /base_controller/autostart: True
 * /base_controller/type: robot_controllers...
 * /bellows_controller/autostart: True
 * /bellows_controller/controlled_joint: bellows_joint
 * /bellows_controller/mimic_joint: torso_lift_joint
 * /bellows_controller/mimic_scale: 0.5
 * /bellows_controller/type: robot_controllers...
 * /gazebo/bellows_joint/position/d: 0.0
 * /gazebo/bellows_joint/position/i: 0.0
 * /gazebo/bellows_joint/position/i_clamp: 0.0
 * /gazebo/bellows_joint/position/p: 10.0
 * /gazebo/bellows_joint/velocity/d: 0.0
 * /gazebo/bellows_joint/velocity/i: 0.0
 * /gazebo/bellows_joint/velocity/i_clamp: 0.0
 * /gazebo/bellows_joint/velocity/p: 25.0
 * /gazebo/default_controllers: ['arm_controller/...
 * /gazebo/elbow_flex_joint/position/d: 0.1
 * /gazebo/elbow_flex_joint/position/i: 0.0
 * /gazebo/elbow_flex_joint/position/i_clamp: 0.0
 * /gazebo/elbow_flex_joint/position/p: 100.0
 * /gazebo/elbow_flex_joint/velocity/d: 0.0
 * /gazebo/elbow_flex_joint/velocity/i: 0.0
 * /gazebo/elbow_flex_joint/velocity/i_clamp: 0.0
 * /gazebo/elbow_flex_joint/velocity/p: 150.0
 * /gazebo/enable_ros_network: True
 * /gazebo/forearm_roll_joint/position/d: 0.1
 * /gazebo/forearm_roll_joint/position/i: 0.0
 * /gazebo/forearm_roll_joint/position/i_clamp: 0.0
 * /gazebo/forearm_roll_joint/position/p: 100.0
 * /gazebo/forearm_roll_joint/velocity/d: 0.0
 * /gazebo/forearm_roll_joint/velocity/i: 0.0
 * /gazebo/forearm_roll_joint/velocity/i_clamp: 0.0
 * /gazebo/forearm_roll_joint/velocity/p: 150.0
 * /gazebo/head_pan_joint/position/d: 0.0
 * /gazebo/head_pan_joint/position/i: 0.0
 * /gazebo/head_pan_joint/position/i_clamp: 0.0
 * /gazebo/head_pan_joint/position/p: 2.0
 * /gazebo/head_pan_joint/velocity/d: 0.0
 * /gazebo/head_pan_joint/velocity/i: 0.0
 * /gazebo/head_pan_joint/velocity/i_clamp: 0.0
 * /gazebo/head_pan_joint/velocity/p: 2.0
 * /gazebo/head_tilt_joint/position/d: 0.0
 * /gazebo/head_tilt_joint/position/i: 0.0
 * /gazebo/head_tilt_joint/position/i_clamp: 0.0
 * /gazebo/head_tilt_joint/position/p: 10.0
 * /gazebo/head_tilt_joint/velocity/d: 0.0
 * /gazebo/head_tilt_joint/velocity/i: 0.0
 * /gazebo/head_tilt_joint/velocity/i_clamp: 0.0
 * /gazebo/head_tilt_joint/velocity/p: 3.0
 * /gazebo/l_gripper_finger_joint/position/d: 0.0
 * /gazebo/l_gripper_finger_joint/position/i: 0.0
 * /gazebo/l_gripper_finger_joint/position/i_clamp: 0.0
 * /gazebo/l_gripper_finger_joint/position/p: 5000.0
 * /gazebo/l_gripper_finger_joint/velocity/d: 0.0
 * /gazebo/l_gripper_finger_joint/velocity/i: 0.0
 * /gazebo/l_gripper_finger_joint/velocity/i_clamp: 0.0
 * /gazebo/l_gripper_finger_joint/velocity/p: 0.0
 * /gazebo/l_wheel_joint/position/d: 0.0
 * /gazebo/l_wheel_joint/position/i: 0.0
 * /gazebo/l_wheel_joint/position/i_clamp: 0.0
 * /gazebo/l_wheel_joint/position/p: 0.0
 * /gazebo/l_wheel_joint/velocity/d: 0.0
 * /gazebo/l_wheel_joint/velocity/i: 0.5
 * /gazebo/l_wheel_joint/velocity/i_clamp: 6.0
 * /gazebo/l_wheel_joint/velocity/p: 8.85
 * /gazebo/r_gripper_finger_joint/position/d: 0.0
 * /gazebo/r_gripper_finger_joint/position/i: 0.0
 * /gazebo/r_gripper_finger_joint/position/i_clamp: 0.0
 * /gazebo/r_gripper_finger_joint/position/p: 5000.0
 * /gazebo/r_gripper_finger_joint/velocity/d: 0.0
 * /gazebo/r_gripper_finger_joint/velocity/i: 0.0
 * /gazebo/r_gripper_finger_joint/velocity/i_clamp: 0.0
 * /gazebo/r_gripper_finger_joint/velocity/p: 0.0
 * /gazebo/r_wheel_joint/position/d: 0.0
 * /gazebo/r_wheel_joint/position/i: 0.0
 * /gazebo/r_wheel_joint/position/i_clamp: 0.0
 * /gazebo/r_wheel_joint/position/p: 0.0
 * /gazebo/r_wheel_joint/velocity/d: 0.0
 * /gazebo/r_wheel_joint/velocity/i: 0.5
 * /gazebo/r_wheel_joint/velocity/i_clamp: 6.0
 * /gazebo/r_wheel_joint/velocity/p: 8.85
 * /gazebo/shoulder_lift_joint/position/d: 0.1
 * /gazebo/shoulder_lift_joint/position/i: 0.0
 * /gazebo/shoulder_lift_joint/position/i_clamp: 0.0
 * /gazebo/shoulder_lift_joint/position/p: 100.0
 * /gazebo/shoulder_lift_joint/velocity/d: 0.0
 * /gazebo/shoulder_lift_joint/velocity/i: 0.0
 * /gazebo/shoulder_lift_joint/velocity/i_clamp: 0.0
 * /gazebo/shoulder_lift_joint/velocity/p: 200.0
 * /gazebo/shoulder_pan_joint/position/d: 0.1
 * /gazebo/shoulder_pan_joint/position/i: 0.0
 * /gazebo/shoulder_pan_joint/position/i_clamp: 0.0
 * /gazebo/shoulder_pan_joint/position/p: 100.0
 * /gazebo/shoulder_pan_joint/velocity/d: 0.0
 * /gazebo/shoulder_pan_joint/velocity/i: 2.0
 * /gazebo/shoulder_pan_joint/velocity/i_clamp: 1.0
 * /gazebo/shoulder_pan_joint/velocity/p: 200.0
 * /gazebo/torso_lift_joint/position/d: 0.0
 * /gazebo/torso_lift_joint/position/i: 0.0
 * /gazebo/torso_lift_joint/position/i_clamp: 0.0
 * /gazebo/torso_lift_joint/position/p: 1000.0
 * /gazebo/torso_lift_joint/velocity/d: 0.0
 * /gazebo/torso_lift_joint/velocity/i: 0.0
 * /gazebo/torso_lift_joint/velocity/i_clamp: 0.0
 * /gazebo/torso_lift_joint/velocity/p: 100000.0
 * /gazebo/upperarm_roll_joint/position/d: 0.1
 * /gazebo/upperarm_roll_joint/position/i: 0.0
 * /gazebo/upperarm_roll_joint/position/i_clamp: 0.0
 * /gazebo/upperarm_roll_joint/position/p: 100.0
 * /gazebo/upperarm_roll_joint/velocity/d: 0.0
 * /gazebo/upperarm_roll_joint/velocity/i: 0.0
 * /gazebo/upperarm_roll_joint/velocity/i_clamp: 0.0
 * /gazebo/upperarm_roll_joint/velocity/p: 10.0
 * /gazebo/wrist_flex_joint/position/d: 0.1
 * /gazebo/wrist_flex_joint/position/i: 0.0
 * /gazebo/wrist_flex_joint/position/i_clamp: 0.0
 * /gazebo/wrist_flex_joint/position/p: 100.0
 * /gazebo/wrist_flex_joint/velocity/d: 0.0
 * /gazebo/wrist_flex_joint/velocity/i: 0.0
 * /gazebo/wrist_flex_joint/velocity/i_clamp: 0.0
 * /gazebo/wrist_flex_joint/velocity/p: 100.0
 * /gazebo/wrist_roll_joint/position/d: 0.1
 * /gazebo/wrist_roll_joint/position/i: 0.0
 * /gazebo/wrist_roll_joint/position/i_clamp: 0.0
 * /gazebo/wrist_roll_joint/position/p: 100.0
 * /gazebo/wrist_roll_joint/velocity/d: 0.0
 * /gazebo/wrist_roll_joint/velocity/i: 0.0
 * /gazebo/wrist_roll_joint/velocity/i_clamp: 0.0
 * /gazebo/wrist_roll_joint/velocity/p: 100.0
 * /gripper_controller/gripper_action/centering/d: 0.0
 * /gripper_controller/gripper_action/centering/i: 0.0
 * /gripper_controller/gripper_action/centering/i_clamp: 0.0
 * /gripper_controller/gripper_action/centering/p: 1000.0
 * /gripper_controller/gripper_action/type: robot_controllers...
 * /head_camera/crop_decimate/decimation_x: 4
 * /head_camera/crop_decimate/decimation_y: 4
 * /head_camera/crop_decimate/queue_size: 1
 * /head_camera/head_camera_nodelet_manager/num_worker_threads: 2
 * /head_controller/follow_joint_trajectory/joints: ['head_pan_joint'...
 * /head_controller/follow_joint_trajectory/type: robot_controllers...
 * /head_controller/point_head/type: robot_controllers...
 * /publish_base_scan_raw/lazy: True
 * /robot/serial: ABCDEFGHIJKLMNOPQ...
 * /robot/version: 0.0.1
 * /robot_description: <?xml version="1....
 * /robot_state_publisher/publish_frequency: 100.0
 * /rosdistro: melodic
 * /rosversion: 1.14.10
 * /torso_controller/follow_joint_trajectory/joints: ['torso_lift_joint']
 * /torso_controller/follow_joint_trajectory/type: robot_controllers...
 * /use_sim_time: True

NODES
  /
    cmd_vel_mux (topic_tools/mux)
    gazebo (gazebo_ros/gzserver)
    gazebo_gui (gazebo_ros/gzclient)
    prepare_robot (fetch_gazebo/prepare_simulated_robot.py)
    publish_base_scan_raw (topic_tools/relay)
    robot_state_publisher (robot_state_publisher/robot_state_publisher)
    urdf_spawner (gazebo_ros/spawn_model)
  /head_camera/
    crop_decimate (nodelet/nodelet)
    head_camera_nodelet_manager (nodelet/nodelet)
  /head_camera/depth_downsample/
    points_downsample (nodelet/nodelet)

auto-starting new master
process[master]: started with pid [3766]
ROS_MASTER_URI=http://localhost:11311

setting /run_id to 504a1e90-33d7-11eb-ac21-04ed33eed89c
process[rosout-1]: started with pid [3777]
started core service [/rosout]
process[gazebo-2]: started with pid [3784]
process[gazebo_gui-3]: started with pid [3789]
process[robot_state_publisher-4]: started with pid [3794]
process[urdf_spawner-5]: started with pid [3795]
process[prepare_robot-6]: started with pid [3800]
[ WARN] [WallTime: 1606828543.422825604] [node:/robot_state_publisher] [func:treeFromUrdfModel]: The root link base_link has an inertia specified in the URDF, but KDL does not support a root link with an inertia.  As a workaround, you can add an extra dummy link to your URDF.
process[head_camera/head_camera_nodelet_manager-7]: started with pid [3802]
process[head_camera/crop_decimate-8]: started with pid [3804]
process[head_camera/depth_downsample/points_downsample-9]: started with pid [3807]
[ INFO] [WallTime: 1606828543.459413047] [node:/head_camera/crop_decimate] [func:loadNodelet]: Loading nodelet /head_camera/crop_decimate of type image_proc/crop_decimate to manager /head_camera/head_camera_nodelet_manager with the following remappings:
[ INFO] [WallTime: 1606828543.460719433] [node:/head_camera/crop_decimate] [func:loadNodelet]: /head_camera/camera/camera_info -> /head_camera/depth_registered/camera_info
[ INFO] [WallTime: 1606828543.460755334] [node:/head_camera/crop_decimate] [func:loadNodelet]: /head_camera/camera/image_raw -> /head_camera/depth_registered/image_raw
[ INFO] [WallTime: 1606828543.460774344] [node:/head_camera/crop_decimate] [func:loadNodelet]: /head_camera/camera_out -> /head_camera/depth_downsample
[ INFO] [WallTime: 1606828543.463059705] [node:/head_camera/crop_decimate] [func:service::exists]: waitForService: Service [/head_camera/head_camera_nodelet_manager/load_nodelet] has not been advertised, waiting...
process[publish_base_scan_raw-10]: started with pid [3815]
[ INFO] [WallTime: 1606828543.487320064] [node:/head_camera/depth_downsample/points_downsample] [func:loadNodelet]: Loading nodelet /head_camera/depth_downsample/points_downsample of type depth_image_proc/point_cloud_xyz to manager /head_camera/head_camera_nodelet_manager with the following remappings:
process[cmd_vel_mux-11]: started with pid [3820]
[ INFO] [WallTime: 1606828543.492815842] [node:/head_camera/depth_downsample/points_downsample] [func:loadNodelet]: /head_camera/depth_downsample/image_rect -> /head_camera/depth_downsample/image_raw
[ INFO] [WallTime: 1606828543.494966318] [node:/head_camera/depth_downsample/points_downsample] [func:service::exists]: waitForService: Service [/head_camera/head_camera_nodelet_manager/load_nodelet] has not been advertised, waiting...
[ INFO] [WallTime: 1606828543.501866786] [node:/head_camera/head_camera_nodelet_manager] [func:Loader::Impl::advertiseRosApi]: Initializing nodelet with 2 worker threads.
[ INFO] [WallTime: 1606828543.506323517] [node:/head_camera/crop_decimate] [func:service::waitForService]: waitForService: Service [/head_camera/head_camera_nodelet_manager/load_nodelet] is now available.
[ INFO] [WallTime: 1606828543.516495719] [node:/head_camera/depth_downsample/points_downsample] [func:service::waitForService]: waitForService: Service [/head_camera/head_camera_nodelet_manager/load_nodelet] is now available.
[ INFO] [WallTime: 1606828543.781663982] [node:/gazebo] [func:GazeboRosApiPlugin::Load]: Finished loading Gazebo ROS API Plugin.
[ INFO] [WallTime: 1606828543.783198124] [node:/gazebo] [func:service::exists]: waitForService: Service [/gazebo/set_physics_properties] has not been advertised, waiting...
[ INFO] [WallTime: 1606828543.856049680] [node:/gazebo_gui] [func:GazeboRosApiPlugin::Load]: Finished loading Gazebo ROS API Plugin.
[ INFO] [WallTime: 1606828543.857064815] [node:/gazebo_gui] [func:service::exists]: waitForService: Service [/gazebo_gui/set_physics_properties] has not been advertised, waiting...
[INFO] [WallTime: 1606828544.366230, 0.000000] [node:/urdf_spawner] [func:SpawnModelNode.run]: Loading model XML from ros parameter robot_description
[INFO] [WallTime: 1606828544.381295, 0.000000] [node:/urdf_spawner] [func:spawn_urdf_model_client]: Waiting for service /gazebo/spawn_urdf_model
Warning [parser.cc:626] Can not find the XML attribute 'version' in sdf XML tag for model: demo_cube. Please specify the SDF protocol supported in the model configuration file. The first sdf tag in the config file will be used 
[ INFO] [WallTime: 1606828544.497885538] [node:/gazebo] [func:service::waitForService]: waitForService: Service [/gazebo/set_physics_properties] is now available.
[ INFO] [WallTime: 1606828544.514546962] [node:/gazebo] [func:GazeboRosApiPlugin::physicsReconfigureThread]: Physics dynamic reconfigure ready.
[INFO] [WallTime: 1606828544.684319, 0.142000] [node:/urdf_spawner] [func:spawn_urdf_model_client]: Calling service /gazebo/spawn_urdf_model
[ INFO] [WallTime: 1606828545.132726209, 0.333000000] [node:/gazebo] [func:__cxx11::string gazebo::GetRobotNamespace]: Laser Plugin: Using the 'robotNamespace' param: '/'
[ INFO] [WallTime: 1606828545.132784387, 0.333000000] [node:/gazebo] [func:GazeboRosLaser::Load]: Starting Laser Plugin (ns = /)
[ INFO] [WallTime: 1606828545.133576753, 0.333000000] [node:/gazebo] [func:GazeboRosLaser::LoadThread]: Laser Plugin (ns = /)  <tf_prefix_>, set to ""
[INFO] [WallTime: 1606828545.191140, 0.333000] [node:/urdf_spawner] [func:spawn_urdf_model_client]: Spawn status: SpawnModel: Successfully spawned entity
[ INFO] [WallTime: 1606828545.336729592, 0.333000000] [node:/gazebo] [func:__cxx11::string gazebo::GetRobotNamespace]: Camera Plugin: Using the 'robotNamespace' param: '/'
[ INFO] [WallTime: 1606828545.338805349, 0.333000000] [node:/gazebo] [func:GazeboRosCameraUtils::LoadThread]: Camera Plugin (ns = /)  <tf_prefix_>, set to ""
[urdf_spawner-5] process has finished cleanly
log file: /home/sktometometo/.ros/log/504a1e90-33d7-11eb-ac21-04ed33eed89c/urdf_spawner-5*.log
[ WARN] [WallTime: 1606828545.830403748, 0.333000000] [node:/gazebo] [func:treeFromUrdfModel]: The root link base_link has an inertia specified in the URDF, but KDL does not support a root link with an inertia.  As a workaround, you can add an extra dummy link to your URDF.
[ INFO] [WallTime: 1606828545.832453198, 0.333000000] [node:/gazebo] [func:ControllerManager::requestStart]: Started arm_controller/gravity_compensation
[ INFO] [WallTime: 1606828545.845240266, 0.333000000] [node:/gazebo] [func:ControllerManager::requestStart]: Started base_controller
[ WARN] [WallTime: 1606828545.859168307, 0.333000000] [node:/gazebo] [func:treeFromUrdfModel]: The root link base_link has an inertia specified in the URDF, but KDL does not support a root link with an inertia.  As a workaround, you can add an extra dummy link to your URDF.
[ INFO] [WallTime: 1606828545.880689428, 0.333000000] [node:/gazebo] [func:ControllerManager::requestStart]: Started bellows_controller
[ INFO] [WallTime: 1606828545.886735007, 0.333000000] [node:/gazebo] [func:FetchGazeboPlugin::Init]: Finished initializing FetchGazeboPlugin
[ INFO] [WallTime: 1606828546.476321140, 0.923000000] [node:/gazebo] [func:ControllerManager::requestStart]: Started head_controller/follow_joint_trajectory
[ INFO] [WallTime: 1606828546.476447608, 0.923000000] [node:/gazebo] [func:ControllerManager::requestStart]: Started arm_controller/follow_joint_trajectory
[ INFO] [WallTime: 1606828546.476572344, 0.923000000] [node:/gazebo] [func:ControllerManager::requestStart]: Started gripper_controller/gripper_action
[prepare_robot-6] process has finished cleanly
log file: /home/sktometometo/.ros/log/504a1e90-33d7-11eb-ac21-04ed33eed89c/prepare_robot-6*.log
[ INFO] [WallTime: 1606828604.557254216, 51.594000000] [node:/gazebo] [func:ControllerManager::requestStart]: Stopped head_controller/follow_joint_trajectory
[ INFO] [WallTime: 1606828604.557355111, 51.594000000] [node:/gazebo] [func:ControllerManager::requestStart]: Started head_controller/point_head
[ INFO] [WallTime: 1606829042.394880280, 367.428000000] [node:/gazebo] [func:ControllerManager::requestStart]: Started torso_controller/follow_joint_trajectory
```

```bash
~ $ roslaunch fetch_gazebo_demo demo.launch 
... logging to /home/sktometometo/.ros/log/504a1e90-33d7-11eb-ac21-04ed33eed89c/roslaunch-Persing-4373.log
Checking log directory for disk usage. This may take a while.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://Persing:45431/

SUMMARY
========

PARAMETERS
 * /amcl/base_frame_id: base_link
 * /amcl/global_frame_id: map
 * /amcl/gui_publish_rate: 10.0
 * /amcl/kld_err: 0.01
 * /amcl/kld_z: 0.99
 * /amcl/laser_likelihood_max_dist: 2.0
 * /amcl/laser_max_beams: 30
 * /amcl/laser_max_range: -1.0
 * /amcl/laser_min_range: -1.0
 * /amcl/laser_model_type: likelihood_field
 * /amcl/laser_sigma_hit: 0.2
 * /amcl/laser_z_hit: 0.95
 * /amcl/laser_z_rand: 0.05
 * /amcl/max_particles: 2000
 * /amcl/min_particles: 500
 * /amcl/odom_alpha1: 0.15
 * /amcl/odom_alpha2: 0.5
 * /amcl/odom_alpha3: 0.5
 * /amcl/odom_alpha4: 0.15
 * /amcl/odom_frame_id: odom
 * /amcl/odom_model_type: diff-corrected
 * /amcl/recovery_alpha_fast: 0.0
 * /amcl/recovery_alpha_slow: 0.0
 * /amcl/resample_interval: 2
 * /amcl/save_pose_rate: 0.5
 * /amcl/transform_tolerance: 0.5
 * /amcl/update_min_a: 0.25
 * /amcl/update_min_d: 0.1
 * /amcl/use_map_topic: False
 * /basic_grasping_perception/gripper/approach/desired: 0.15
 * /basic_grasping_perception/gripper/approach/min: 0.145
 * /basic_grasping_perception/gripper/finger_depth: 0.02
 * /basic_grasping_perception/gripper/gripper_tolerance: 0.05
 * /basic_grasping_perception/gripper/retreat/desired: 0.15
 * /basic_grasping_perception/gripper/retreat/min: 0.145
 * /basic_grasping_perception/gripper/tool_to_planning_frame: 0.18
 * /move_base/NavfnROS/allow_unknown: True
 * /move_base/NavfnROS/default_tolerance: 0.0
 * /move_base/NavfnROS/planner_window_x: 0.0
 * /move_base/NavfnROS/planner_window_y: 0.0
 * /move_base/NavfnROS/visualize_potential: False
 * /move_base/TrajectoryPlannerROS/acc_lim_theta: 0.8
 * /move_base/TrajectoryPlannerROS/acc_lim_x: 0.35
 * /move_base/TrajectoryPlannerROS/acc_lim_y: 0.0
 * /move_base/TrajectoryPlannerROS/angular_sim_granularity: 0.025
 * /move_base/TrajectoryPlannerROS/dwa: False
 * /move_base/TrajectoryPlannerROS/gdist_scale: 12.0
 * /move_base/TrajectoryPlannerROS/heading_lookahead: 0.325
 * /move_base/TrajectoryPlannerROS/heading_scoring: True
 * /move_base/TrajectoryPlannerROS/heading_scoring_timestep: 0.8
 * /move_base/TrajectoryPlannerROS/holonomic_robot: False
 * /move_base/TrajectoryPlannerROS/latch_xy_goal_tolerance: True
 * /move_base/TrajectoryPlannerROS/max_vel_theta: 1.5
 * /move_base/TrajectoryPlannerROS/max_vel_x: 1.0
 * /move_base/TrajectoryPlannerROS/max_vel_y: 0.0
 * /move_base/TrajectoryPlannerROS/meter_scoring: True
 * /move_base/TrajectoryPlannerROS/min_in_place_vel_theta: 0.3
 * /move_base/TrajectoryPlannerROS/min_vel_theta: -1.5
 * /move_base/TrajectoryPlannerROS/min_vel_x: 0.15
 * /move_base/TrajectoryPlannerROS/min_vel_y: 0.0
 * /move_base/TrajectoryPlannerROS/occdist_scale: 0.1
 * /move_base/TrajectoryPlannerROS/oscillation_reset_dist: 0.05
 * /move_base/TrajectoryPlannerROS/pdist_scale: 8.0
 * /move_base/TrajectoryPlannerROS/publish_cost_grid_pc: False
 * /move_base/TrajectoryPlannerROS/sim_granularity: 0.025
 * /move_base/TrajectoryPlannerROS/sim_time: 1.0
 * /move_base/TrajectoryPlannerROS/vtheta_samples: 10
 * /move_base/TrajectoryPlannerROS/vx_samples: 3
 * /move_base/TrajectoryPlannerROS/xy_goal_tolerance: 0.1
 * /move_base/TrajectoryPlannerROS/yaw_goal_tolerance: 0.1
 * /move_base/aggressive_reset/reset_distance: 0.5
 * /move_base/base_global_planner: navfn/NavfnROS
 * /move_base/base_local_planner: base_local_planne...
 * /move_base/conservative_reset/reset_distance: 3.0
 * /move_base/controller_frequency: 25.0
 * /move_base/controller_patience: 15.0
 * /move_base/global_costmap/global_frame: map
 * /move_base/global_costmap/inflater/inflation_radius: 0.7
 * /move_base/global_costmap/inflater/robot_radius: 0.3
 * /move_base/global_costmap/obstacles/base_scan/clearing: True
 * /move_base/global_costmap/obstacles/base_scan/data_type: LaserScan
 * /move_base/global_costmap/obstacles/base_scan/marking: True
 * /move_base/global_costmap/obstacles/base_scan/max_obstacle_height: 0.3
 * /move_base/global_costmap/obstacles/base_scan/min_obstacle_height: 0.25
 * /move_base/global_costmap/obstacles/base_scan/obstacle_range: 4.0
 * /move_base/global_costmap/obstacles/base_scan/raytrace_range: 5.0
 * /move_base/global_costmap/obstacles/base_scan/topic: base_scan
 * /move_base/global_costmap/obstacles/max_obstacle_height: 2.0
 * /move_base/global_costmap/obstacles/observation_sources: base_scan
 * /move_base/global_costmap/obstacles/z_resolution: 0.125
 * /move_base/global_costmap/obstacles/z_voxels: 16
 * /move_base/global_costmap/plugins: [{'type': 'costma...
 * /move_base/global_costmap/publish_frequency: 0.0
 * /move_base/global_costmap/robot_base_frame: base_link
 * /move_base/global_costmap/robot_radius: 0.3
 * /move_base/global_costmap/static_map: True
 * /move_base/global_costmap/transform_tolerance: 0.5
 * /move_base/global_costmap/update_frequency: 5.0
 * /move_base/local_costmap/global_frame: odom
 * /move_base/local_costmap/height: 4.0
 * /move_base/local_costmap/inflater/inflation_radius: 0.7
 * /move_base/local_costmap/inflater/robot_radius: 0.3
 * /move_base/local_costmap/obstacles/base_scan/clearing: True
 * /move_base/local_costmap/obstacles/base_scan/data_type: LaserScan
 * /move_base/local_costmap/obstacles/base_scan/marking: True
 * /move_base/local_costmap/obstacles/base_scan/max_obstacle_height: 0.3
 * /move_base/local_costmap/obstacles/base_scan/min_obstacle_height: 0.25
 * /move_base/local_costmap/obstacles/base_scan/obstacle_range: 4.0
 * /move_base/local_costmap/obstacles/base_scan/raytrace_range: 5.0
 * /move_base/local_costmap/obstacles/base_scan/topic: base_scan
 * /move_base/local_costmap/obstacles/max_obstacle_height: 2.0
 * /move_base/local_costmap/obstacles/observation_sources: base_scan
 * /move_base/local_costmap/obstacles/publish_observations: False
 * /move_base/local_costmap/obstacles/z_resolution: 0.125
 * /move_base/local_costmap/obstacles/z_voxels: 16
 * /move_base/local_costmap/plugins: [{'type': 'costma...
 * /move_base/local_costmap/publish_frequency: 2.0
 * /move_base/local_costmap/resolution: 0.025
 * /move_base/local_costmap/robot_base_frame: base_link
 * /move_base/local_costmap/robot_radius: 0.3
 * /move_base/local_costmap/rolling_window: True
 * /move_base/local_costmap/transform_tolerance: 0.5
 * /move_base/local_costmap/width: 4.0
 * /move_base/oscillation_distance: 0.5
 * /move_base/oscillation_timeout: 10.0
 * /move_base/planner_frequency: 0.0
 * /move_base/planner_patience: 5.0
 * /move_base/recovery_behavior_enabled: True
 * /move_base/recovery_behaviors: [{'type': 'clear_...
 * /move_base/rotate_recovery/frequency: 20.0
 * /move_base/rotate_recovery/sim_granularity: 0.017
 * /move_group/allow_trajectory_execution: True
 * /move_group/allowed_execution_duration_scaling: 1.2
 * /move_group/allowed_goal_duration_margin: 0.5
 * /move_group/arm/longest_valid_segment_fraction: 0.005
 * /move_group/arm/planner_configs: ['SBLkConfigDefau...
 * /move_group/arm/projection_evaluator: joints(shoulder_p...
 * /move_group/arm_with_torso/longest_valid_segment_fraction: 0.05
 * /move_group/arm_with_torso/planner_configs: ['SBLkConfigDefau...
 * /move_group/arm_with_torso/projection_evaluator: joints(torso_lift...
 * /move_group/capabilities: move_group/MoveGr...
 * /move_group/controller_list: [{'default': True...
 * /move_group/gripper/planner_configs: ['SBLkConfigDefau...
 * /move_group/jiggle_fraction: 0.05
 * /move_group/max_safe_path_cost: 1
 * /move_group/moveit_controller_manager: moveit_simple_con...
 * /move_group/moveit_manage_controllers: True
 * /move_group/planner_configs/BKPIECEkConfigDefault/type: geometric::BKPIECE
 * /move_group/planner_configs/ESTkConfigDefault/type: geometric::EST
 * /move_group/planner_configs/KPIECEkConfigDefault/type: geometric::KPIECE
 * /move_group/planner_configs/LBKPIECEkConfigDefault/type: geometric::LBKPIECE
 * /move_group/planner_configs/PRMkConfigDefault/type: geometric::PRM
 * /move_group/planner_configs/PRMstarkConfigDefault/type: geometric::PRMstar
 * /move_group/planner_configs/RRTConnectkConfigDefault/type: geometric::RRTCon...
 * /move_group/planner_configs/RRTkConfigDefault/type: geometric::RRT
 * /move_group/planner_configs/RRTstarkConfigDefault/type: geometric::RRTstar
 * /move_group/planner_configs/SBLkConfigDefault/type: geometric::SBL
 * /move_group/planner_configs/TRRTkConfigDefault/type: geometric::TRRT
 * /move_group/planning_plugin: ompl_interface/OM...
 * /move_group/planning_scene_monitor/publish_geometry_updates: True
 * /move_group/planning_scene_monitor/publish_planning_scene: True
 * /move_group/planning_scene_monitor/publish_state_updates: True
 * /move_group/planning_scene_monitor/publish_transforms_updates: True
 * /move_group/request_adapters: default_planner_r...
 * /move_group/start_state_max_bounds_error: 0.1
 * /robot_description_kinematics/arm/kinematics_solver: fetch_arm/IKFastK...
 * /robot_description_kinematics/arm/kinematics_solver_attempts: 3
 * /robot_description_kinematics/arm/kinematics_solver_search_resolution: 0.005
 * /robot_description_kinematics/arm/kinematics_solver_timeout: 0.005
 * /robot_description_kinematics/arm_with_torso/kinematics_solver: kdl_kinematics_pl...
 * /robot_description_kinematics/arm_with_torso/kinematics_solver_attempts: 3
 * /robot_description_kinematics/arm_with_torso/kinematics_solver_search_resolution: 0.005
 * /robot_description_kinematics/arm_with_torso/kinematics_solver_timeout: 0.005
 * /robot_description_planning/joint_limits/elbow_flex_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/elbow_flex_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/elbow_flex_joint/max_acceleration: 1.5
 * /robot_description_planning/joint_limits/elbow_flex_joint/max_velocity: 1.5
 * /robot_description_planning/joint_limits/forearm_roll_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/forearm_roll_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/forearm_roll_joint/max_acceleration: 1.5
 * /robot_description_planning/joint_limits/forearm_roll_joint/max_velocity: 1.57
 * /robot_description_planning/joint_limits/left_gripper_joint/has_acceleration_limits: False
 * /robot_description_planning/joint_limits/left_gripper_joint/has_velocity_limits: False
 * /robot_description_planning/joint_limits/left_gripper_joint/max_acceleration: 0
 * /robot_description_planning/joint_limits/left_gripper_joint/max_velocity: 0
 * /robot_description_planning/joint_limits/right_gripper_joint/has_acceleration_limits: False
 * /robot_description_planning/joint_limits/right_gripper_joint/has_velocity_limits: False
 * /robot_description_planning/joint_limits/right_gripper_joint/max_acceleration: 0
 * /robot_description_planning/joint_limits/right_gripper_joint/max_velocity: 0
 * /robot_description_planning/joint_limits/shoulder_lift_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/shoulder_lift_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/shoulder_lift_joint/max_acceleration: 1.0
 * /robot_description_planning/joint_limits/shoulder_lift_joint/max_velocity: 1.0
 * /robot_description_planning/joint_limits/shoulder_pan_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/shoulder_pan_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/shoulder_pan_joint/max_acceleration: 1.5
 * /robot_description_planning/joint_limits/shoulder_pan_joint/max_velocity: 1.256
 * /robot_description_planning/joint_limits/torso_lift_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/torso_lift_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/torso_lift_joint/max_acceleration: 0.5
 * /robot_description_planning/joint_limits/torso_lift_joint/max_velocity: 0.1
 * /robot_description_planning/joint_limits/upperarm_roll_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/upperarm_roll_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/upperarm_roll_joint/max_acceleration: 1.5
 * /robot_description_planning/joint_limits/upperarm_roll_joint/max_velocity: 1.57
 * /robot_description_planning/joint_limits/wrist_flex_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/wrist_flex_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/wrist_flex_joint/max_acceleration: 2.5
 * /robot_description_planning/joint_limits/wrist_flex_joint/max_velocity: 2.26
 * /robot_description_planning/joint_limits/wrist_roll_joint/has_acceleration_limits: True
 * /robot_description_planning/joint_limits/wrist_roll_joint/has_velocity_limits: True
 * /robot_description_planning/joint_limits/wrist_roll_joint/max_acceleration: 2.5
 * /robot_description_planning/joint_limits/wrist_roll_joint/max_velocity: 2.26
 * /robot_description_semantic: <?xml version="1....
 * /rosdistro: melodic
 * /rosversion: 1.14.10

NODES
  /
    amcl (amcl/amcl)
    basic_grasping_perception (simple_grasping/basic_grasping_perception)
    demo (fetch_gazebo_demo/demo.py)
    map_server (map_server/map_server)
    move_base (move_base/move_base)
    move_group (moveit_ros_move_group/move_group)
    tilt_head_node (fetch_navigation/tilt_head.py)

ROS_MASTER_URI=http://localhost:11311

running rosparam delete /move_group/sensors
ERROR: parameter [/move_group/sensors] is not set
process[map_server-1]: started with pid [4398]
process[amcl-2]: started with pid [4399]
process[move_base-3]: started with pid [4400]
process[tilt_head_node-4]: started with pid [4406]
process[move_group-5]: started with pid [4415]
process[basic_grasping_perception-6]: started with pid [4418]
process[demo-7]: started with pid [4420]
[ WARN] [WallTime: 1606828596.888583374] [node:/amcl] [func:requestMap]: Request for map failed; trying again...
[ INFO] [WallTime: 1606828596.930680090] [node:/move_group] [func:core::RobotModel::buildModel]: Loading robot model 'fetch'...
[ INFO] [WallTime: 1606828596.933259524] [node:/move_group] [func:core::JointModel* moveit::core::RobotModel::constructJointModel]: No root/virtual joint specified in SRDF. Assuming fixed joint
[ WARN] [WallTime: 1606828597.046173471, 46.300000000] [node:/move_base] [func:Costmap2DROS::Costmap2DROS]: Timed out waiting for transform from base_link to map to become available before running costmap, tf error: canTransform: target_frame map does not exist.. canTransform returned after 46.3 timeout was 0.1.
[ WARN] [WallTime: 1606828597.168020636, 46.386000000] [node:/move_group] [func:core::SolverAllocatorFn kinematics_plugin_loader::KinematicsPluginLoader::getLoaderFunction]: Kinematics solver doesn't support #attempts anymore, but only a timeout.
Please remove the parameter '/robot_description_kinematics/arm/kinematics_solver_attempts' from your configuration.
[ WARN] [WallTime: 1606828597.202335940, 46.413000000] [node:/move_group] [func:treeFromUrdfModel]: The root link base_link has an inertia specified in the URDF, but KDL does not support a root link with an inertia.  As a workaround, you can add an extra dummy link to your URDF.
[ INFO] [WallTime: 1606828597.530208236, 46.585000000] [node:/move_group] [func:PlanningSceneMonitor::startPublishingPlanningScene]: Publishing maintained planning scene on 'monitored_planning_scene'
[ INFO] [WallTime: 1606828597.535487512, 46.589000000] [node:/move_group] [func:main]: MoveGroup debug mode is ON
Starting planning scene monitors...
[ INFO] [WallTime: 1606828597.535544537, 46.589000000] [node:/move_group] [func:PlanningSceneMonitor::startSceneMonitor]: Starting planning scene monitor
[ INFO] [WallTime: 1606828597.541462918, 46.590000000] [node:/move_group] [func:PlanningSceneMonitor::startSceneMonitor]: Listening to '/planning_scene'
[ INFO] [WallTime: 1606828597.541575870, 46.590000000] [node:/move_group] [func:PlanningSceneMonitor::startWorldGeometryMonitor]: Starting world geometry update monitor for collision objects, attached objects, octomap updates.
[ INFO] [WallTime: 1606828597.544982982, 46.590000000] [node:/move_group] [func:PlanningSceneMonitor::startWorldGeometryMonitor]: Listening to '/collision_object'
[ INFO] [WallTime: 1606828597.547871406, 46.590000000] [node:/move_group] [func:PlanningSceneMonitor::startWorldGeometryMonitor]: Listening to '/planning_scene_world' for planning scene world geometry
[ WARN] [WallTime: 1606828597.548844323, 46.590000000] [node:/move_group] [func:OccupancyMapMonitor::initialize]: Resolution not specified for Octomap. Assuming resolution = 0.1 instead
[ERROR] [WallTime: 1606828597.550081573, 46.590000000] [node:/move_group] [func:OccupancyMapMonitor::initialize]: Failed to find 3D sensor plugin parameters for octomap generation
[ INFO] [WallTime: 1606828597.627750911, 46.657000000] [node:/move_group] [func:PlanningSceneMonitor::startStateMonitor]: Listening to '/attached_collision_object' for attached collision objects
Planning scene monitors started.
[ INFO] [WallTime: 1606828597.672661763, 46.690000000] [node:/move_group] [func:OMPLInterface::OMPLInterface]: Initializing OMPL interface using ROS parameters
[ INFO] [WallTime: 1606828597.733526052, 46.727000000] [node:/move_group] [func:PlanningPipeline::configure]: Using planning interface 'OMPL'
[ INFO] [WallTime: 1606828597.740205284, 46.733000000] [node:/move_group] [func:FixWorkspaceBounds::FixWorkspaceBounds]: Param 'default_workspace_bounds' was not set. Using default value: 10
[ INFO] [WallTime: 1606828597.741003912, 46.734000000] [node:/move_group] [func:FixStartStateBounds::FixStartStateBounds]: Param 'start_state_max_bounds_error' was set to 0.1
[ INFO] [WallTime: 1606828597.741709966, 46.734000000] [node:/move_group] [func:FixStartStateBounds::FixStartStateBounds]: Param 'start_state_max_dt' was not set. Using default value: 0.5
[ INFO] [WallTime: 1606828597.742516494, 46.735000000] [node:/move_group] [func:FixStartStateCollision::FixStartStateCollision]: Param 'start_state_max_dt' was not set. Using default value: 0.5
[ INFO] [WallTime: 1606828597.743430325, 46.736000000] [node:/move_group] [func:FixStartStateCollision::FixStartStateCollision]: Param 'jiggle_fraction' was set to 0.05
[ INFO] [WallTime: 1606828597.743974296, 46.736000000] [node:/move_group] [func:FixStartStateCollision::FixStartStateCollision]: Param 'max_sampling_attempts' was not set. Using default value: 100
[ INFO] [WallTime: 1606828597.744070517, 46.736000000] [node:/move_group] [func:PlanningPipeline::configure]: Using planning request adapter 'Add Time Parameterization'
[ INFO] [WallTime: 1606828597.744114144, 46.736000000] [node:/move_group] [func:PlanningPipeline::configure]: Using planning request adapter 'Fix Workspace Bounds'
[ INFO] [WallTime: 1606828597.744143679, 46.736000000] [node:/move_group] [func:PlanningPipeline::configure]: Using planning request adapter 'Fix Start State Bounds'
[ INFO] [WallTime: 1606828597.744198211, 46.736000000] [node:/move_group] [func:PlanningPipeline::configure]: Using planning request adapter 'Fix Start State In Collision'
[ INFO] [WallTime: 1606828597.744267045, 46.736000000] [node:/move_group] [func:PlanningPipeline::configure]: Using planning request adapter 'Fix Start State Path Constraints'
[ WARN] [WallTime: 1606828598.028218554, 46.935000000] [node:/move_base] [func:Costmap2DROS::checkOldParam]: global_costmap: Pre-Hydro parameter "static_map" unused since "plugins" is provided
[ INFO] [WallTime: 1606828598.029963245, 46.936000000] [node:/move_base] [func:Costmap2DROS::Costmap2DROS]: global_costmap: Using plugin "static_map"
[ INFO] [WallTime: 1606828598.040967690, 46.944000000] [node:/move_base] [func:StaticLayer::onInitialize]: Requesting the map...
[ INFO] [WallTime: 1606828598.049378664, 46.952000000] [node:/move_group] [func:MoveItSimpleControllerManager::MoveItSimpleControllerManager]: Added FollowJointTrajectory controller for arm_controller
[ INFO] [WallTime: 1606828598.199046341, 47.049000000] [node:/move_base] [func:StaticLayer::incomingMap]: Resizing costmap to 401 X 444 at 0.050000 m/pix
[ INFO] [WallTime: 1606828598.352474572, 47.137000000] [node:/move_group] [func:MoveItSimpleControllerManager::MoveItSimpleControllerManager]: Added FollowJointTrajectory controller for arm_with_torso_controller
[ INFO] [WallTime: 1606828598.370376849, 47.149000000] [node:/move_base] [func:StaticLayer::onInitialize]: Received a 401 X 444 map at 0.050000 m/pix
[ INFO] [WallTime: 1606828598.375655296, 47.152000000] [node:/move_base] [func:Costmap2DROS::Costmap2DROS]: global_costmap: Using plugin "obstacles"
[ INFO] [WallTime: 1606828598.563646082, 47.258000000] [node:/move_group] [func:MoveItSimpleControllerManager::MoveItSimpleControllerManager]: Added GripperCommand controller for gripper_controller
[ INFO] [WallTime: 1606828598.563833105, 47.258000000] [node:/move_group] [func:MoveItSimpleControllerManager::getControllersList]: Returned 3 controllers in list
[ INFO] [WallTime: 1606828598.573310124, 47.265000000] [node:/move_base] [func:ObstacleLayer::onInitialize]:     Subscribed to Topics: base_scan
[ INFO] [WallTime: 1606828598.603163506, 47.285000000] [node:/move_group] [func:TrajectoryExecutionManager::initialize]: Trajectory execution is managing controllers
Loading 'move_group/ApplyPlanningSceneService'...
Loading 'move_group/ClearOctomapService'...
Loading 'move_group/MoveGroupCartesianPathService'...
Loading 'move_group/MoveGroupExecuteTrajectoryAction'...
[ INFO] [WallTime: 1606828598.685161495, 47.331000000] [node:/move_base] [func:Costmap2DROS::Costmap2DROS]: global_costmap: Using plugin "inflater"
Loading 'move_group/MoveGroupGetPlanningSceneService'...
Loading 'move_group/MoveGroupKinematicsService'...
Loading 'move_group/MoveGroupMoveAction'...
Loading 'move_group/MoveGroupPickPlaceAction'...
Loading 'move_group/MoveGroupPlanService'...
Loading 'move_group/MoveGroupQueryPlannersService'...
Loading 'move_group/MoveGroupStateValidationService'...
[ INFO] [WallTime: 1606828598.810317226, 47.416000000] [node:/move_group] [func:MoveGroupExe::configureCapabilities]: 

********************************************************
* MoveGroup using: 
*     - ApplyPlanningSceneService
*     - ClearOctomapService
*     - CartesianPathService
*     - ExecuteTrajectoryAction
*     - GetPlanningSceneService
*     - KinematicsService
*     - MoveAction
*     - PickPlaceAction
*     - MotionPlanService
*     - QueryPlannersService
*     - StateValidationService
********************************************************

[ INFO] [WallTime: 1606828598.810472166, 47.416000000] [node:/move_group] [func:MoveGroupContext::status]: MoveGroup context using planning plugin ompl_interface/OMPLPlanner
[ INFO] [WallTime: 1606828598.810531710, 47.416000000] [node:/move_group] [func:MoveGroupContext::status]: MoveGroup context initialization complete

You can start planning now!

[ INFO] [WallTime: 1606828598.833878900, 47.432000000] [node:/move_base] [func:Costmap2DROS::Costmap2DROS]: local_costmap: Using plugin "obstacles"
[ INFO] [WallTime: 1606828598.840428639, 47.435000000] [node:/move_base] [func:ObstacleLayer::onInitialize]:     Subscribed to Topics: base_scan
[ INFO] [WallTime: 1606828598.917091866, 47.482000000] [node:/move_base] [func:Costmap2DROS::Costmap2DROS]: local_costmap: Using plugin "inflater"
[ INFO] [WallTime: 1606828598.990388534, 47.529000000] [node:/move_base] [func:MoveBase::MoveBase]: Created local_planner base_local_planner/TrajectoryPlannerROS
[ INFO] [WallTime: 1606828599.003441515, 47.539000000] [node:/move_base] [func:TrajectoryPlannerROS::initialize]: Sim period is set to 0.04
[ WARN] [WallTime: 1606828599.010334165, 47.544000000] [node:/move_base] [func:loadParameterWithDeprecation]: Parameter pdist_scale is deprecated. Please use the name path_distance_bias instead.
[ WARN] [WallTime: 1606828599.011897912, 47.545000000] [node:/move_base] [func:loadParameterWithDeprecation]: Parameter gdist_scale is deprecated. Please use the name goal_distance_bias instead.
[ INFO] [WallTime: 1606828599.550666759, 47.933000000] [node:/move_base] [func:ClearCostmapRecovery::initialize]: Recovery behavior will clear layer 'obstacles'
[ INFO] [WallTime: 1606828599.559889194, 47.940000000] [node:/move_base] [func:ClearCostmapRecovery::initialize]: Recovery behavior will clear layer 'obstacles'
[ INFO] [WallTime: 1606828599.621481991, 47.980000000] [node:/move_base] [func:OdometryHelperRos::odomCallback]: odom received!
[INFO] [WallTime: 1606828601.511732, 49.340000] [node:/demo] [func:MoveBaseClient.__init__]: Waiting for move_base...
[INFO] [WallTime: 1606828601.790860, 49.561000] [node:/demo] [func:FollowTrajectoryClient.__init__]: Waiting for torso_controller...
[INFO] [WallTime: 1606828602.093564, 49.766000] [node:/demo] [func:PointHeadClient.__init__]: Waiting for head_controller...
[INFO] [WallTime: 1606828602.382999, 49.982000] [node:/demo] [func:PlanningSceneInterface.__init__]: Waiting for get_planning_scene
[INFO] [WallTime: 1606828602.392725, 49.988000] [node:/demo] [func:PickPlaceInterface.__init__]: Connecting to pickup action...
[INFO] [WallTime: 1606828602.688209, 50.211000] [node:/demo] [func:PickPlaceInterface.__init__]: ...connected
[INFO] [WallTime: 1606828602.690381, 50.213000] [node:/demo] [func:PickPlaceInterface.__init__]: Connecting to place action...
[INFO] [WallTime: 1606828602.990166, 50.432000] [node:/demo] [func:PickPlaceInterface.__init__]: ...connected
[INFO] [WallTime: 1606828603.295561, 50.659000] [node:/demo] [func:GraspingClient.__init__]: Waiting for basic_grasping_perception/find_objects...
[INFO] [WallTime: 1606828603.589035, 50.876000] [node:/demo] [func:<module>]: Moving to table...
[ WARN] [WallTime: 1606828624.444793113, 66.638000000] [node:/move_base] [func:ClearCostmapRecovery::runBehavior]: Clearing both costmaps to unstuck robot (3.00m).
[ WARN] [WallTime: 1606828637.976593027, 76.718000000] [node:/move_base] [func:RotateRecovery::runBehavior]: Rotate recovery behavior started.
[ WARN] [WallTime: 1606828660.695720706, 93.149000000] [node:/move_base] [func:ClearCostmapRecovery::runBehavior]: Clearing both costmaps to unstuck robot (0.50m).
[ERROR] [WallTime: 1606828674.225478740, 103.228000000] [node:/move_base] [func:MoveBase::executeCycle]: Aborting because the robot appears to be oscillating over and over. Even after executing all recovery behaviors
[ WARN] [WallTime: 1606828681.966613750, 109.073000000] [node:/move_base] [func:ClearCostmapRecovery::runBehavior]: Clearing both costmaps to unstuck robot (3.00m).
[ WARN] [WallTime: 1606828689.030139727, 114.111000000] [node:/move_base] [func:RotateRecovery::runBehavior]: Rotate recovery behavior started.
[ WARN] [WallTime: 1606828712.522178676, 130.503000000] [node:/move_base] [func:ClearCostmapRecovery::runBehavior]: Clearing both costmaps to unstuck robot (3.00m).
[ WARN] [WallTime: 1606828725.050462616, 138.981000000] [node:/move_base] [func:RotateRecovery::runBehavior]: Rotate recovery behavior started.
[ WARN] [WallTime: 1606828748.938910745, 155.271000000] [node:/move_base] [func:ClearCostmapRecovery::runBehavior]: Clearing both costmaps to unstuck robot (3.00m).
[ WARN] [WallTime: 1606828756.841809105, 160.311000000] [node:/move_base] [func:RotateRecovery::runBehavior]: Rotate recovery behavior started.
[ WARN] [WallTime: 1606828777.261143266, 174.261000000] [node:/move_base] [func:ClearCostmapRecovery::runBehavior]: Clearing both costmaps to unstuck robot (3.00m).
[ WARN] [WallTime: 1606828784.667815867, 179.301000000] [node:/move_base] [func:RotateRecovery::runBehavior]: Rotate recovery behavior started.
[ WARN] [WallTime: 1606828805.493947365, 195.181000000] [node:/move_base] [func:ClearCostmapRecovery::runBehavior]: Clearing both costmaps to unstuck robot (3.00m).
[ WARN] [WallTime: 1606828818.451555211, 205.261000000] [node:/move_base] [func:RotateRecovery::runBehavior]: Rotate recovery behavior started.
[ WARN] [WallTime: 1606828833.952878050, 216.651000000] [node:/move_base] [func:ClearCostmapRecovery::runBehavior]: Clearing both costmaps to unstuck robot (0.50m).
[ WARN] [WallTime: 1606828848.432822172, 226.731000000] [node:/move_base] [func:ClearCostmapRecovery::runBehavior]: Clearing both costmaps to unstuck robot (3.00m).
[ WARN] [WallTime: 1606828874.704644934, 245.611000000] [node:/move_base] [func:RotateRecovery::runBehavior]: Rotate recovery behavior started.
[ WARN] [WallTime: 1606828895.816367368, 260.063000000] [node:/move_base] [func:ClearCostmapRecovery::runBehavior]: Clearing both costmaps to unstuck robot (3.00m).
[ WARN] [WallTime: 1606828903.665284646, 265.101000000] [node:/move_base] [func:RotateRecovery::runBehavior]: Rotate recovery behavior started.
[ WARN] [WallTime: 1606828927.045389512, 281.031000000] [node:/move_base] [func:ClearCostmapRecovery::runBehavior]: Clearing both costmaps to unstuck robot (3.00m).
[ WARN] [WallTime: 1606828952.642950339, 299.871000000] [node:/move_base] [func:RotateRecovery::runBehavior]: Rotate recovery behavior started.
[ WARN] [WallTime: 1606828973.164265990, 315.961000000] [node:/move_base] [func:ClearCostmapRecovery::runBehavior]: Clearing both costmaps to unstuck robot (3.00m).
[ WARN] [WallTime: 1606828986.144639118, 326.041000000] [node:/move_base] [func:RotateRecovery::runBehavior]: Rotate recovery behavior started.
[ WARN] [WallTime: 1606829007.400543745, 342.521000000] [node:/move_base] [func:ClearCostmapRecovery::runBehavior]: Clearing both costmaps to unstuck robot (0.50m).
[ERROR] [WallTime: 1606829042.358124593, 367.401000000] [node:/move_base] [func:MoveBase::executeCycle]: Aborting because a valid control could not be found. Even after executing all recovery behaviors
[INFO] [WallTime: 1606829042.392132, 367.425000] [node:/demo] [func:<module>]: Raising torso...
[INFO] [WallTime: 1606829050.088133, 373.459000] [node:/demo] [func:<module>]: Picking object...
Traceback (most recent call last):
  File "/opt/ros/melodic/lib/fetch_gazebo_demo/demo.py", line 274, in <module>
    grasping_client.updateScene()
  File "/opt/ros/melodic/lib/fetch_gazebo_demo/demo.py", line 152, in updateScene
    wait = False)
TypeError: addSolidPrimitive() got an unexpected keyword argument 'wait'
[demo-7] process has died [pid 4420, exit code 1, cmd /opt/ros/melodic/lib/fetch_gazebo_demo/demo.py __name:=demo __log:=/home/sktometometo/.ros/log/504a1e90-33d7-11eb-ac21-04ed33eed89c/demo-7.log].
log file: /home/sktometometo/.ros/log/504a1e90-33d7-11eb-ac21-04ed33eed89c/demo-7*.log
```

**Expected behavior**
It is expected that fetch will achieve the demo without any error.

**Screenshots**
Please see above.

**catkin workspace (please complete the following information):**
 - fetch_gazebo version: 0.9.2 
 - Ubuntu version: 18.04
 - ROS version: melodic
 - current catkin workspace information `wstool info`: no catkin workspace

**Additional context**
Add any other context about the problem here.

--- a/fetch_gazebo_demo/scripts/demo.py
+++ b/fetch_gazebo_demo/scripts/demo.py
@@ -136,9 +136,9 @@ class GraspingClient(object):
 
         # remove previous objects
         for name in self.scene.getKnownCollisionObjects():
-            self.scene.removeCollisionObject(name, False)
+            self.scene.removeCollisionObject(name, use_service=False)
         for name in self.scene.getKnownAttachedObjects():
-            self.scene.removeAttachedObject(name, False)
+            self.scene.removeAttachedObject(name, use_service=False)
         self.scene.waitForSync()
 
         # insert objects to scene
@@ -149,7 +149,7 @@ class GraspingClient(object):
             self.scene.addSolidPrimitive(obj.object.name,
                                          obj.object.primitives[0],
                                          obj.object.primitive_poses[0],
-                                         wait = False)
+                                         use_service=False)
 
         for obj in find_result.support_surfaces:
             # extend surface to floor, and make wider since we have narrow field of view
@@ -163,8 +163,7 @@ class GraspingClient(object):
             self.scene.addSolidPrimitive(obj.name,
                                          obj.primitives[0],
                                          obj.primitive_poses[0],
-                                         wait = False)
-
+                                         use_service=False)
         self.scene.waitForSync()
 
         # store for grasping

--- a/fetch_gazebo_demo/scripts/pick_place_demo.py
+++ b/fetch_gazebo_demo/scripts/pick_place_demo.py
@@ -151,7 +151,7 @@ class GraspingClient(object):
             self.scene.addSolidPrimitive(obj.object.name,
                                          obj.object.primitives[0],
                                          obj.object.primitive_poses[0],
-                                         use_service = False)
+                                         use_service=False)
             if obj.object.primitive_poses[0].position.x < 0.85:
                 objects.append([obj, obj.object.primitive_poses[0].position.z])
 
@@ -167,8 +167,7 @@ class GraspingClient(object):
             self.scene.addSolidPrimitive(obj.name,
                                          obj.primitives[0],
                                          obj.primitive_poses[0],
-                                         use_service = True
-                                         )
+                                         use_service=False)
 
         self.scene.waitForSync()
 

